gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/avatarpicker/AvatarPicker.kt

```diff
@@ -154,75 +163,23 @@ internal fun AvatarPicker(uiState: AvatarPickerUiState, onAvatarSelected: (Avata
     }
 }
 
-@Composable
-private fun AvatarsSection(
-    avatars: List<AvatarUi>,
-    onAvatarSelected: (Avatar) -> Unit,
-    modifier: Modifier = Modifier,
-) {
-    var popupVisible by rememberSaveable { mutableStateOf(false) }
-    var popupYOffset by rememberSaveable { mutableIntStateOf(0) }
-    Box(
-        modifier = modifier
-            .border(
-                width = 1.dp,
-                color = MaterialTheme.colorScheme.surfaceContainerHighest,
-                shape = RoundedCornerShape(8.dp),
-            )
-            .padding(16.dp),
-    ) {
-        Column {
-            Text(
-                text = stringResource(id = R.string.avatar_picker_title),
-                fontSize = 22.sp,
-                fontWeight = FontWeight.Bold,
-            )
-            Text(
-                text = stringResource(R.string.avatar_picker_description),
-                fontSize = 15.sp,
-                color = MaterialTheme.colorScheme.tertiary,
-                modifier = Modifier.padding(top = 4.dp),
-            )
+private const val UCROP_COMPRESSION_QUALITY = 100
 
-            LazyRow(horizontalArrangement = Arrangement.spacedBy(8.dp), modifier = Modifier.padding(vertical = 24.dp)) {
-                items(items = avatars, key = { it.avatarId }) { avatarModel ->
-                    when (avatarModel) {
-                        is AvatarUi.Uploaded -> SelectableAvatar(
-                            imageUrl = avatarModel.avatar.fullUrl,
-                            isSelected = avatarModel.isSelected,
-                            isLoading = avatarModel.isLoading,
-                            onAvatarClicked = {
-                                onAvatarSelected(avatarModel.avatar)
-                            },
-                            modifier = Modifier.size(96.dp),
-                        )
-                    }
-                }
-            }
-            Button(
-                modifier = Modifier
-                    .fillMaxWidth()
-                    .onPlaced { layoutCoordinates -> popupYOffset = layoutCoordinates.size.height },
-                onClick = { popupVisible = true },
-                shape = RoundedCornerShape(4.dp),
-                contentPadding = PaddingValues(14.dp),
-            ) {
-                Text(
-                    text = stringResource(R.string.avatar_picker_upload_image),
-                    style = MaterialTheme.typography.titleMedium,
-                )
-            }
-        }
-        if (popupVisible) {
-            MediaPickerPopup(
-                alignment = Alignment.BottomCenter,
-                onDismissRequest = { popupVisible = false },
-                offset = IntOffset(0, -popupYOffset - 30),
-                onChoosePhotoClick = { popupVisible = false },
-                onTakePhotoClick = { popupVisible = false },
-            )
-        }
+private fun ActivityResultLauncher<Intent>.launchAvatarCrop(image: Uri, tempFile: File, context: Context) {
```

Comment:
This could be in its own file along with the `UCROP_COMPRESSION_QUALITY` constant